ui/src/app/(index)/_components/Announcements.tsx

```diff
@@ -0,0 +1,28 @@
+import { Alert, AlertDescription, AlertTitle } from '@/components/ui/alert';
+import { AlertCircle } from 'lucide-react';
+import { getAnnouncements } from '@/services/GroupingsApiService';
+
+const Announcements = async () => {
+    const announcements = await getAnnouncements()
+    const activeAnnouncements = () => {
+        if (!announcements || !announcements.announcements) {
+            return []
+        }
+        return announcements.announcements.filter((announcement) => announcement.state === 'Active')
+            .map((announcement) => announcement.message)
+    }
+    return (
+        <div className="mt-5 mb-5">
+            {activeAnnouncements().map((announcement:string, index:number) => (
+                <Alert key={index} className="bg-yellow-100 border border-yellow-200 mb-2">
+                    <AlertCircle aria-label="icon" className="h-4 w-4"/>
+                    <AlertTitle className="font-semibold">Announcement</AlertTitle>
+                    <AlertDescription>
+                        {announcement}
+                    </AlertDescription>
+                </Alert>))}
+        </div>
+    )
```

Comment:
Reformat to:
``
const { announcements } = await getAnnouncements();

const activeAnnouncements = () => {
    if (!announcements) {
        return []
    }
    return announcements
        .filter((announcement) => announcement.state === 'Active')
        .map((announcement) => announcement.message)
};

return (
        <div className="mt-5 mb-5">
            {activeAnnouncements().map((announcement: string, index: number) => (
                <Alert key={index} className="bg-yellow-100 border border-yellow-200 mb-2">
                    <AlertCircle aria-label="icon" className="h-4 w-4"/>
                    <AlertTitle className="font-semibold">Announcement</AlertTitle>
                    <AlertDescription>
                        {announcement}
                    </AlertDescription>
                </Alert>))}
        </div>
    );
```
Make sure to put space-out chunks of code for clarity.